EarablesKIT/EarablesKIT/EarablesKIT/Models/Extentionmodel/Activities/RunningActivity/RunningEventArgs.cs

```diff
@@ -6,5 +6,6 @@ namespace EarablesKIT.Models.Extentionmodel.Activities.RunningActivity
 {
     class RunningEventArgs : ActivityArgs
     {
+
```

Comment:
hier fehlt das Running attribut.
public bool Running;
        public RunningEventArgs(bool isRunning)
        {
            this.Running = isRunning;
        }

Reply:
Ist mir klar, ist aber nicht meine Aufgabe :D 
Hab da ja auch eigentlich nix geändert

Reply:
doch die Zeile hinzugefügt, deshalb hatte ich merge-komflikte und fragte mich warum du die datei angefasst hast :D